bpfman/src/utils.rs

```diff
@@ -81,6 +81,14 @@ pub(crate) fn bytes_to_u32(bytes: Vec<u8>) -> u32 {
     )
 }
 
+pub(crate) fn bytes_to_u16(bytes: Vec<u8>) -> u16 {
+    u16::from_ne_bytes(
+        bytes
+            .try_into()
+            .expect("unable to martial &[u8] to &[u8; 4]"),
```

Comment:
Just saw this one, it should be `Marshall` not `Martial`

Reply:
Good catch :)